main.html

```diff
@@ -1,77 +1,231 @@
-<p>A really simple example:</p>
-<p>You can write paragraphs like this.</p>
-<pre data-code-language="python" data-executable="true" data-type="programlisting">hey = &quot;hello world&quot;, 38+4
-print(hey hello. This is an executable code block )!
-</pre>
-<div id="the-id">
-<p>This is a block inside a div with an id the-id.
-Very cool. Yes.</p>
-</div>
-<pre data-code-language="" >hey = &quot;hello world&quot;, 38+4
-print(hey hello. This is a non executable block )!
-</pre>
-<div id="another-id">
-<h2>More complicated example</h2>
-</div>
-<pre data-code-language="python" data-executable="true" data-type="programlisting">x = [1,1]
-for i in range(12):
-    x.append(x[-1] + x[-2]) 
-print(', '.join(str(y) for y in x))
-</pre>
-<h2>A Chart</h2>
-<div id="this-example">
-</div>
-<pre data-code-language="python" data-executable="true" data-type="programlisting">%matplotlib inline
-import matplotlib.pyplot as plt
-import numpy as np
-x = np.linspace(0, 3*np.pi, 500)
-plt.plot(x, np.sin(x**2))
-plt.title('A simple chirp');
-</pre>
-<h2>Seaborn Example</h2>
-<pre data-code-language="python" data-executable="true" data-type="programlisting">from __future__ import print_function, division
-
-%matplotlib inline
-import matplotlib.pyplot as plt
-import numpy as np
-import pandas as pd
-
-x = np.linspace(0, 10, 1000)
-plt.plot(x, np.sin(x), x, np.cos(x));
-
-import seaborn as sns
-sns.set()
-plt.plot(x, np.sin(x), x, np.cos(x));
-</pre>
-<h2>Interactive widgets</h2>
-<pre data-code-language="python" data-executable="true" data-type="programlisting">%matplotlib inline
-from IPython.display import Image
-from IPython.html.widgets import interact
-from numpy import pi, cos, sin
-import numpy as np
-import pylab as plt
-import seaborn as sns
-sns.set()
-</pre>
-<pre data-code-language="pyt" data-executable="true" data-type="programlisting">def plot_fringe(bl_length, wavelength):
-    &quot;&quot;&quot; Plot the fringe function for a baseline (see Fig 1)
-
-    bl_length:      distance between antennas, in m
-    wavelength:     wavelength, in m
-    &quot;&quot;&quot;
-    theta = np.linspace(-np.pi, np.pi, 401)
-    l = sin(theta)
-    F = cos(2 * pi * bl_length * l / wavelength)
-    F2 = cos(2 * pi * 2 * bl_length * l / wavelength)
-
-    plt.plot(l, F, c='#cc0000', label=&quot;Baseline 1-2&quot;)
-    plt.plot(l, F2, c='#0000cc', label=&quot;Baseline 1-3&quot;)
-    plt.xlabel(&quot;$sin(\\theta)$&quot;)
-    plt.ylabel(&quot;Fringe amplitude&quot;)
-    plt.ylim(-2, 2)
-    plt.legend()
-
-f = interact(plot_fringe, bl_length=(1, 100), wavelength=(1, 100))
-&lt;/pre&gt;
-</pre>
+<h2>Introduction</h2>
+
+<p>
+  InnerSource is the methodology of applying open source practices and principles to internal software development.
+  This methodology results in shared software that is responsive and nimble to the changing needs of its many internal consumers.
+</p>
+
+<p>
+  In this Oriole, we'll define some key terms used when discussing InnerSource.
```

Comment:
I think that these three sentences are a bit too generic.  They don't have a lot of 'teeth'  It is too self-referential.
Suggestions:

Change "define some key terms used when discussing InnerSource" to "teach you how to identify scenarios where InnerSource Principles can be applied"

Add something like: "Be able to describe what InnerSource looks like"

Reply:
> Change "define some key terms

🤔 Defining key terms is an explicit goal of this content.  Do you agree with [these proposed goals](https://github.com/rrrutledge/innersource-intro/blob/master/outline.md) for this project?  I want to check if we have different goals in mind as we comment and work on this material together.